HIRS_Utils/src/test/java/hirs/validation/SupplyChainCredentialValidatorTest.java

```diff
@@ -1992,6 +1995,211 @@ public final void testValPlatCredentialAttributesV2p0RequiredComponentNoSerialOr
                 SupplyChainCredentialValidator.PLATFORM_ATTRIBUTES_VALID);
     }
 
+    /**
+     * Tests that SupplyChainCredentialValidator passes with a base and delta certificate where
```

Comment:
Would these unit tests cover the scenarios listed in the "Test" section in the "Test patterns for Platform Certificate v1.1" doc?

Reply:
I didn't know how many @iadgovuser26 wanted, so I just did one to show the case of a base with deltas validating against a paccor string.

Reply:
I spoke with him briefly about this and he did comment about wanting more system tests with hard files.  I'll add one more unit test to exercise failure code.